distributed/tests/test_worker.py

```diff
@@ -2456,8 +2456,8 @@ async def test_forget_dependents_after_release(c, s, a):
     assert fut2.key not in {d.key for d in a.tasks[fut.key].dependents}
 
 
-@gen_cluster(client=True, nthreads=[("", 1)] * 2, timeout=5000000)
-async def test_steak_during_task_deserialization(c, s, a, b, monkeypatch):
+@gen_cluster(client=True, nthreads=[("", 1)] * 2)
```

Comment:
Since two workers is the default, we don't need to specify `nthreads=` (unless I'm missing something else)

```suggestion
@gen_cluster(client=True)
```